cmd/config.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long: `Modifies Odo specific configuration settings within the config file.
+   # Available Parameters:
+     UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+	Example: fmt.Sprintf("%s\n%s\n",
+		configurationViewCmd.Example,
+		configurationSetCmd.Example),
+	Aliases: []string{"configuration"},
+	// 'odo utils config' is the same as 'odo utils config --help'
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) >= 1 && args[0] != "view" && args[0] != "set" {
+			return fmt.Errorf("Unknown command, use set or view")
+		}
+		return nil
+	}, Run: func(cmd *cobra.Command, args []string) {
+		if len(args) > 0 && args[0] == "set" {
+			configurationSetCmd.Run(cmd, args)
+		} else if len(args) > 0 && args[0] == "view" {
+			configurationViewCmd.Run(cmd, args)
+		} else {
+			cmd.Help()
+		}
+	},
+}
+
+var configurationSetCmd = &cobra.Command{
+	Use:   "set",
+	Short: "Set a value in odo config file",
+	Long: `Set an individual value in the Odo configuration file 
+   # Available Parameters:
+     UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+	Example: `  # Set a configuration value
+    odo utils config set UpdateNotification false`,
```

Comment:
something weird is happening here too. it it 3-spaces not 1-tab, as it is not appearing aligned:

```sh

github.com/redhat-developer/odo  pr_589 ✔                                                                                                                                                                                                                                   3d  ⍉
▶ ./odo utils
Utilities for completion, terminal commands and modifying Odo configurations

Usage:
  odo utils [command]

Examples:
  # Bash autocompletion support
  source <(odo utils completion bash)

  # Zsh autocompletion support
  source <(odo utils completion zsh)

  # Bash terminal PS1 support
  source <(odo utils terminal bash)

  # Zsh terminal PS1 support
  source <(odo utils terminal zsh)

  # Set a configuration value
    odo utils config set UpdateNotification false
  # For viewing the current configuration
    odo utils config view
```